server/local/strings.yml

```diff
@@ -13,5 +13,6 @@ strings: {
   lookAtOtherDashesNoCourses: "It looks like none of your classes are using web sites we know of.  Checkout the ",
   canvasUrl: "https://umich.instructure.com/",
   ctoolsUrl: "https://ctools.umich.edu/portal",
-  noCourses: "You seem to have no courses this term"
+  noCourses: "You seem to have no courses this term",
+  termFailure: "The service we depend on for term information is returning GARBAGE."
```

Comment:
How about?

We are not finding your term information at the moment.  Please try again later.